CHANGELOG.md

```diff
@@ -4,15 +4,105 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v9.2.0](https://github.com/voxpupuli/puppet-zabbix/tree/v9.2.0) (2022-06-03)
+## [v9.2.1](https://github.com/voxpupuli/puppet-zabbix/tree/v9.2.1) (2024-03-14)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-zabbix/compare/v9.2.0...v9.2.1)
+
+**Breaking changes:**
```

Comment:
the next release needs to be a major one we there are breaking changes. please update metadata.json to 10.0.0 and regenerate the changelog.